app/controllers/admin/users_controller.rb

```diff
@@ -15,14 +15,7 @@ def show
 
     def update
       @user = User.find(params[:id])
-      @user.roles = []
-      @user.add_role params[:role]
-      @user.email = params[:user][:email] if params[:user] && params[:user][:email]
-      assignments = params[:assignments].values if params[:assignments]
-      (assignments || []).each do |ass|
-        @user.add_role(ass[:role], Vendor.find(ass[:vendor_id]))
-      end
-      @user.save
+      @user.update(params)
```

Comment:
Can we confirm we're not clobbering some existing method called "update" by doing this?

Reply:
Updated it. Should be fine now.